compiler/fir/diagnostics-diff.delta/README.md

```diff
@@ -0,0 +1,50 @@
+# Diagnostics diff delta tool
+
+This tool extracts data about the differences in diagnostics tests (e.g., `compiler/testData/diagnostics`) for FE 1.0 versus FIR, at two
+different commits, and computes the delta between the two sets of data. The purpose of this tool is to show the impact of a set of changes
+(between the two commits) in getting FIR diagnostics closer to FE 1.0 diagnostics. For example, this tool may be used to see whether
+your changes got all the diagnostics that you expect to be reported, or if you missed some cases.
+
+## Invoking the tool
+
+To install the tool:
+```shell script
+./gradlew :compiler:fir:diagnostics-diff.delta:installDist
+```
+
+The executable can be found at `compiler/fir/diagnostics-diff-delta/build/install/diagnostics-diff-delta/bin/diagnostics-diff-delta`. Run it
```

Comment:
Looks like the path should be `compiler/fir/diagnostics-diff.delta/...`

Reply:
Thanks for catching that. The other README is wrong too. 🙂  Fixed.